integration/js/utils/WebdriverSauceLabs.js

```diff
@@ -200,8 +188,17 @@ class SaucelabsSession {
 
   async getSessionId() {
     if (this.sessionId === undefined) {
-      const session = await this.driver.getSession();
-      this.sessionId = session.getId();
+      const cap = await this.driver.getCapabilities();
+      const url = cap.get('testobject_test_report_api_url')
+      if(url !== undefined)
```

Comment:
Can we add a comment here that say this is only for mobile test from the old test object framework?

Reply:
Definitely agree, added!